CMakeLists.txt

```diff
@@ -72,7 +75,7 @@ set(CURSES_NEED_NCURSES TRUE)
 # Dependencies
 find_package(FS REQUIRED)
 find_package(Numa)
-find_package(LLVM 5.0.0)
```

Comment:
@jfrohnhofen - This does not cause and tests to fail, even if we use LLVM 6. Is this valid?

Reply:
It may very well be. I would expect the LLVM apis to be pretty stable.
If this causes any trouble in the following PRs or in the future in general, we should adapt/migrate the Jitting to LLVM 6 anyway.

Reply:
Yes, the files `CloneFunction.cpp` and `InlineFunction.cpp` from JIT PR 5 #813 will not compile with LLVM6 as they use unofficial apis from LLVM5.

Reply:
Good thing that we notice that now. @FabianWiebe - do you already have enough insight to see what needs to change?

Reply:
Yes, we need to take the new versions for LLVM6 of `CloneFunction.cpp` and `InlineFunction.cpp` from https://github.com/llvm-mirror/llvm/tree/release_60/lib/Transforms/Utils and adjust them to Hyrise, again.

Reply:
Could you get started on that, please?

Reply:
Yes.

Reply:
@mrks @FabianWiebe Do you intend to support multiple LLVM versions for jitting in parallel? That sound like maintenance hell to me...
If we require LLVM 6 for jitting moving forward `find_package(LLVM 6.0.0 CONFIG)` should be added and the CI systems would need to be upgraded to LLVM 6.

Reply:
Will do.

Reply:
@jfrohnhofen No, I would only support one.